test/Stats.unittest.js

```diff
@@ -110,12 +133,21 @@ describe(
 					},
 					compiler: {
 						context: ""
+					},
+					hooks: {
+						stats: new SyncHook(["stats"])
 					}
 				});
 				const obj = mockStats.toJson();
 				obj.errors[0].should.be.equal("firstError");
 			});
 		});
+		// ?????????????????????????????????????????????????????????????????????????????????? //
```

Comment:
Bad rebase?

Reply:
Yeah, will edit that, sorry. :smiley:, wanted to ask question related to mocking.